app/controllers/orders_controller.rb

```diff
@@ -1,7 +1,32 @@
 class OrdersController < ApplicationController
-
+  before_action :require_login, only: [:show]
+  before_action :get_order, only: [:show]
+  
   def cart
     @session = session
   end
 
+  def show
+    # require_login controller filter: only authorized merchant can view
```

Comment:
This will make an order not viewable by folks who are looking up an order but aren't logged in -- I don't know if we discussed how we want that feature to work, maybe check in with Yieni since she's working on the look_up feature?

Reply:
Yes, I talked to Yieni while working on this feature. Given the user stories and the different requirements, we will be splitting up the two routes/views.

Reply:
Oooo, nice! You're two steps ahead of me! 🤩